libqtile/widget/base.py

```diff
@@ -461,6 +493,26 @@ def _configure(self, qtile, bar):
             self.fontshadow,
             markup=self.markup,
         )
+        if not isinstance(self.scroll_width, int) and self.scroll:
+            logger.warning(f"{self.name}: You must specify a width when enabling scrolling.")
```

Comment:
```suggestion
            logger.warning("%s: You must specify a width when enabling scrolling.", self.name)
```

Reply:
Yes. Need to remember this is not string formatting, this is logger formatting so it's ok to use `%s`!